examples/scale/backend/backend.go

```diff
@@ -81,7 +81,17 @@ func run(cfg config.View) {
 	w := logger.Writer()
 	defer w.Close()
 
-	for {
+	matchesForAssignment := make(chan *pb.Match, 1000)
+	ticketsForDeletion := make(chan string, 1000)
```

Comment:
can we write it in a way without these two channels? it is possible to have these two channels become the bottleneck of the scale test in some scenarios.

Reply:
Bumped up to meet our maximum values for what we'll expect out of the scale tests.  I do want the director to put the breaks on if it can't keep up bough.